### DISCUSSION TITLE
Auto pause doesn't work

### DISCUSSION THREAD

**Discussion by ghost** [d4rken-org/capod#277] [Q&A]
The ear detection works and I have auto pause on but the auto pause doesn't work.

**Comment by d4rken** [comment]
Pods? Version? Phone? Player? :question: :question: :question:

**Comment by dustinmiller18** [comment]
> Pods? Version? Phone? Player? ❓ ❓ ❓

I had the same issue using the Airpods Pro 2 running CAPod v2.12.1-rc0 on a Google Pixel 7a (Android 14) with Spotify.

**Comment by d4rken** [comment]
Probably https://github.com/d4rken-org/capod/issues/38

**Comment by Hosam8895** [comment]
Same issue here , I have samsung s21 ultra and  airpods pro 2 usb c 
Auto play and auto pause are not working

**Comment by xiaoyaomeng** [comment]
很高兴收到您的邮件，看到后将立即回复~Thank you~

**Comment by HK-DDboy** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

**Comment by ohadorpaz** [comment]
Same here.
Samsung galaxy 22, airpods pro Gen 2

**Comment by filonitta** [comment]
If the feature doesn't work, and the issue seems out of scope, then I shouldn't buy the app. At least for now, keep watching the status

**Comment by sohamsalhotra** [comment]
Same issue for me too.
Phone: Pixel 7 Pro / Android 14 Feb 2024 Update
App Version: v2.12.1-rc0
Device: AirPods Pro Gen 2 (USB-C) (Model No. A3048)
Music Source Apps: Multiple (Tested Spotify, YouTube, Google Chrome)
In-Ear status detected on App: Updates both ways within ~0.5s as seen on the App's main status screen (when inserted or removed; each side updates individually)

My current workaround: Audio is paused for each side when placed in the case, or for both sides when either is inserted in the ear and then paused via force sensor (Or, of course, manually pausing from the phone).

Thanks for your work on this project, let me know if me sharing any debug logs would help! :)

**Comment by Doublemyst** [comment]
Same issue, any updates if this is something, which is going to be fixed?
In my case when i take the earbud out, it changes to ambient mode, but in the app it still says that is plugged in, but only when it is in music mode. when i have no music playing and take one earbud out, it recognize it, as one bud out of ear.
